browser/app/js/components/SideBar.js

```diff
@@ -19,58 +19,46 @@ import classNames from 'classnames'
 import ClickOutHandler from 'react-onclickout'
 import Scrollbars from 'react-custom-scrollbars/lib/Scrollbars'
 import connect from 'react-redux/lib/components/connect'
+import logo from '../../img/logo-dark.svg'
 
-import logo from '../../img/logo.svg'
+let SideBar = ({visibleBuckets, loadBucket, currentBucket, selectBucket, searchBuckets, sidebarStatus, clickOutside, showPolicy, storageDetails}) => {
 
-let SideBar = ({visibleBuckets, loadBucket, currentBucket, selectBucket, searchBuckets, sidebarStatus, clickOutside, showPolicy}) => {
-
-  const list = visibleBuckets.map((bucket, i) => {
-    return <li className={ classNames({
-                  'active': bucket === currentBucket
-                }) } key={ i } onClick={ (e) => selectBucket(e, bucket) }>
-             <a href="" className={ classNames({
-                                      'fesli-loading': bucket === loadBucket
-                                    }) }>
-               { bucket }
-             </a>
-             <i className="fesli-trigger" onClick={ showPolicy }></i>
-           </li>
-  })
+    const list = visibleBuckets.map ((bucket, i) => {
+        return  <li className={ classNames ({ 'buckets__list__active': bucket === currentBucket, 'fesli-loading': bucket === loadBucket })  } key={ i } onClick={ (e) => selectBucket (e, bucket) }>
+            <div className="buckets__list__name">{ bucket }</div>
+            <div className="buckets__list__actions">
+                <span>read and write</span>
```

Comment:
It's still in the discussion level. I have added this only for UI purposes. We may add any other information to cover this.